internal/pkg/manifest/validate_test.go

```diff
@@ -332,10 +332,16 @@ func TestLoadBalancedWebService_validate(t *testing.T) {
 					ImageConfig: testImageConfig,
 					Sidecars: map[string]*SidecarConfig{
 						"foo": {
+							Image: Union[*string, ImageLocationOrBuild]{
+								Basic: aws.String("mockImage"),
+							},
```

Comment:
these should use `BasicToUnion()` instead of directly creating the struct so that `isBasic` gets set on the Union struct